src/routes/Books.ts

```diff
@@ -0,0 +1,47 @@
+import Router from '@koa/router';
+import { Book, PostBook } from '../models/Books'
+import db from '../db'
+const bookRouter = new Router();
+const BOOK_ENDPOINT = "/books"
+const ID = "/:id"
+
+bookRouter
+  .post(BOOK_ENDPOINT, async (ctx) => {
+    const data = <PostBook>ctx.request.body
+    const book = new Book();
+    book.name = data.name
+    book.isbn = data.isbn
+    book.author = data.author
+    book.releaseDate = data.releaseDate
+    await db.manager.save(book);
+
+  })
+  .get(BOOK_ENDPOINT, async (ctx) => {
+    const repository = db.getRepository(Book)
+    const books = await repository.find()
+    ctx.body = books;
+  })
+  .get(BOOK_ENDPOINT + ID, async (ctx) => {
+    const repository = db.getRepository(Book)
+    const book = await repository.findOneBy({ uuid: Number(ctx.params.id) })
```

Comment:
404 should be returned as a response in case the book not exists, like in the `PUT` endpoint.